Firestore/core/test/unit/FSTGoogleTestTests.mm

```diff
@@ -214,6 +231,9 @@ void XCTestMethod(XCTestCase* self, SEL _cmd) {
     // Let XCode know that the test ran and succeeded.
     XCTAssertTrue(true);
     return;
+  } else if (result->Skipped()) {
```

Comment:
Unlike the above, this is dependent on the version of Xcode used, so unfortunately, it does need the `#if` on `MAX_ALLOWED` properties.

`XCTSkip` was added on Xcode 11.4, the versions should be (based on https://developer.apple.com/documentation/xcode-release-notes/xcode-11_4-release-notes):

```
__IPHONE_OS_VERSION_MAX_ALLOWED >= 130400 ||
__TV_OS_VERSION_MAX_ALLOWED >= 130400 ||
__MAC_OS_X_VERSION_MAX_ALLOWED >= 101504
```

Reply:
Thank you, I added it.
For some reason, ___MAC_OS_X_VERSION_MAX_ALLOWED_ is on my machine (macOS 11.0 Beta (20A5395g), Xcode 12.1 (12A7403)) defined to ___MAC_10_15_ so I cannot use these guards. However, Availability.h defines ___MAC_10_15_1_ and ___MAC_10_15_4_ as well, and I am building with 10.15 SDK so I do not know where the problem is.

Today I'm going to update the OS so I'll see if it changes anything.

Reply:
With update to `macOS 11.1 Beta (20C5048l)` and `Xcode 12.2 (12B45b)` the SDK was also updated, so now my `__MAC_OS_X_VERSION_MAX_ALLOWED` is defined to `__MAC_11_0`.